quetz/deps.py

```diff
@@ -90,12 +90,17 @@ def get_tasks_worker(
     if config.configured_section("worker"):
         worker = config.worker_type
     else:
-        worker = "thread"
+        worker = "redis-queue"
 
     if worker == "thread":
         worker = ThreadingWorker(background_tasks, dao, auth, session, config)
+        print("Threading Worker")
     elif worker == "subprocess":
         worker = SubprocessWorker(auth.API_key, auth.session)
+        print("Subprocess Worker")
+    elif worker == "redis-queue":
+        worker = RQManager('127.0.0.1', 6379, 0, auth.API_key, auth.session)
```

Comment:
the ip and port of the rq manager should be taken from config, but you can handle it when you finalize the PR

Reply:
Yeah, this is for later.